packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -375,14 +375,26 @@ if (!canUseDOM) {
       headOfPendingCallbacksLinkedList = scheduledCallbackConfig;
       tailOfPendingCallbacksLinkedList = scheduledCallbackConfig;
     } else {
-      // Add latest callback as the new tail of the list
-      scheduledCallbackConfig.prev = tailOfPendingCallbacksLinkedList;
-      // renaming for clarity
-      const oldTailOfPendingCallbacksLinkedList = tailOfPendingCallbacksLinkedList;
-      if (oldTailOfPendingCallbacksLinkedList !== null) {
-        oldTailOfPendingCallbacksLinkedList.next = scheduledCallbackConfig;
+      if (options && options.continued) {
```

Comment:
See other comment. What should happen if `options.continued` is true but we're not inside another callback?